opennms-provision/opennms-provisiond/src/test/java/org/opennms/netmgt/provision/service/ProvisionerIT.java

```diff
@@ -1300,6 +1301,15 @@ public void testProvisionServiceGetScheduleForNodesUponDelete() throws Exception
 
         assertEquals(9, schedulesForNode.size());
         assertEquals(9, m_provisioner.getScheduleLength());
+
+        // Check that re-scheduling won't trigger duplicate scheduling for a given node.
```

Comment:
I'm guessing this is related to NMS-12503? :)

Reply:
yeah, similar issue but https://issues.opennms.org/browse/NMS-12504